packages/patterns/src/types.js

```diff
@@ -524,17 +529,23 @@ export {};
 /** @typedef {(...args: any[]) => any} Method */
 
 /**
- * @template {Record<string | symbol, MethodGuard>} [T=Record<string | symbol, MethodGuard>]
+ * @template {Record<PropertyKey, MethodGuard>} [T=Record<PropertyKey, MethodGuard>]
  * @typedef {{
  *   interfaceName: string,
- *   methodGuards: { [K in keyof T]: K extends symbol ? never : T[K] },
- *   symbolMethodGuards?: CopyMap<(keyof T) & symbol, MethodGuard>,
+ *   methodGuards:
+ *     { [K in Exclude<keyof T, symbol>]: T[K] } &
```

Comment:
This also has a shorter spelling, albeit one that [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) does not claim _official_ support for (which is probably accidental: https://github.com/microsoft/TypeScript-Website/issues/2937 ).
```suggestion
 *     Omit<T, symbol> &
```

Either is fine with me.